### DISCUSSION TITLE
Crosshair ON/OFF toggle with shortcut key

### DISCUSSION THREAD

**Discussion by benclau** [flyinghead/flycast#1175] [Ideas]
Hi,

Would be great if we could be able to toggle crosshair on/off with a shortcut key.

Thank you,

**Comment by flyinghead** [comment]
This has been requested in issue #844

**Comment by benclau** [comment]
Oh, sorry for that. (And I’m the one who asked for it back then… Alzheimer is getting me!)

Thank you,

De : flyinghead ***@***.***>
Envoyé : August 11, 2023 3:26 AM
À : flyinghead/flycast ***@***.***>
Cc : benclau ***@***.***>; Author ***@***.***>
Objet : Re: [flyinghead/flycast] Crosshair ON/OFF toggle with shortcut key (Discussion #1175)


This has been requested in issue #844<https://github.com/flyinghead/flycast/issues/844>
